src/main/java/com/monumental/controllers/MonumentController.java

```diff
@@ -1,47 +1,55 @@
 package com.monumental.controllers;
 
-import com.monumental.exceptions.*;
+import com.monumental.exceptions.ResourceNotFoundException;
 import com.monumental.models.Monument;
+import com.monumental.repositories.MonumentRepository;
 import com.monumental.services.MonumentService;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.*;
 
+import javax.transaction.Transactional;
 import java.util.List;
+import java.util.Optional;
 
 @RestController
+@Transactional
 public class MonumentController {
 
+    @Autowired
+    private MonumentRepository monumentRepository;
+
     @Autowired
     private MonumentService monumentService;
 
     @PostMapping("/api/monument")
     public Monument createMonument(@RequestBody Monument monument) {
-        this.monumentService.insert(monument);
+        this.monumentRepository.save(monument);
         return monument;
     }
 
     @GetMapping("/api/monument/{id}")
     public Monument getMonument(@PathVariable("id") Integer id,
                                 @RequestParam(value = "cascade", defaultValue = "false") Boolean cascade)
             throws ResourceNotFoundException {
-        Monument monument = this.monumentService.get(id, cascade);
+        Optional<Monument> optional = this.monumentRepository.findById(id);
+        if (optional.isEmpty()) throw new ResourceNotFoundException();
+        Monument monument = optional.get();
 
-        if (monument == null) {
-            throw new ResourceNotFoundException();
+        if (cascade) {
+            this.monumentService.initializeAllLazyLoadedCollections(monument);
         }
-
         return monument;
     }
 
     @GetMapping("/api/monuments")
     public List<Monument> getAllMonuments() {
-        return this.monumentService.getAll();
+        return this.monumentRepository.findAll();
```

Comment:
We don't want to expose the ability to get all of the Monuments with lazy loaded collections?

Reply:
We do - the ModelService now has `public fetchAllLazyLoadedCollectitons` exposed which can be called explicitly when necessary

Reply:
This is only necessary for controllers who want all the data to be included in the JSON. Everywhere else, if the method is `@Transactional` (or its class) then it will be able to get whichever fields it needs on the fly just by calling the getter